doc/api/tls.md

```diff
@@ -2,90 +2,138 @@
 
     Stability: 2 - Stable
 
-Use `require('tls')` to access this module.
+The `tls` module provides an implementation of the Transport Layer Security
+(TLS) and Secure Socket Layer (SSL) protocols that is built on top of OpenSSL.
```

Comment:
Yeah, I made a note of that for myself when I was going through this. There are a couple of outdated bits in this that should be updated.